<launch>
  <node name="frame_stabilization_node" pkg="visual_stabilization" type="frame_stabilization_node" output="screen">

    <remap from="undistorted_image" to="/visual_stabilization/image" />
    <remap from="stabilized_image" to="/visual_stabilization/stabilized_image" />
    <remap from="extended_stabilized_image" to="/visual_stabilization/extended_stabilized_image" />

    <!--  General configuration  -->
    <param name="enable_undistortion" value="True"/>
    <param name="enable_saccades" value="True"/>
    <param name="canvas_gain" value="2.0" />
    <param name="record_bag" value="False"/>
    <param name="file_name" value="$(find visual_stabilization)/frame_stabilization_sample.bag"/>
   
    <!--  Dataset configuration  -->
    <rosparam file="$(find visual_stabilization)/config/ecd_dataset_mocap.yaml" />
    <!-- <rosparam file="$(find visual_stabilization)/config/mvsec_indoor_flying.yaml" /> -->
    <!-- <rosparam file="$(find visual_stabilization)/config/vector_image.yaml" /> -->
   
  </node>

  <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" args="--perspective-file $(find visual_stabilization)/rqt/visual_stabilization_frames.perspective" />
</launch>
